stock_operating_unit/model/stock_warehouse.py

```diff
@@ -0,0 +1,59 @@
+# © 2019 Eficent Business and IT Consulting Services S.L.
+# © 2019 Serpent Consulting Services Pvt. Ltd.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class StockWarehouse(models.Model):
+    _inherit = "stock.warehouse"
+
+    def _default_operating_unit(self):
+        if self.company_id:
+            company = self.company_id
+        else:
+            company = self.env["res.company"]._company_default_get("stock.inventory")
```

Comment:
The method '_company_default_get' on res.company is deprecated and shouldn't be used anymore

it's better to use self.env.company

Reply:
Oh, thanks for the info. I didn't know that.
Unfortunately I can't update my PR until the 03.03.2020. I will correct it then.

Reply:
I will make some really heavy tests of this module with a customer with 29 operating units, I already detected some errors in this module, is there a way to take your PR and make a PR to your branch? or I make another PR with your tests and the fixes to bugs I have found?

Reply:
That would be great! You could add my fork as remote, create a branch out of my migration branch and afterwards make a PR to my fork. The second option would be to take my code, update it on your branch and create a new PR.